### DISCUSSION TITLE
Average zones per animal per reserve

### DISCUSSION THREAD

**Discussion by Caldas** [janstehno/cotw-companion#52] [Ideas]
To start a grind we must find all zones but we never know how many are they.

So first we need a counter to zones per animal per reserve.

After the community usage the app will be able to provide an estimate average Zone amount per animal per reserve 🎁

**Comment by janstehno** [comment]
This would be hard to determine when every player has different zones on each map. I could show a number of located zones but that would not correspond to the reality. Also, I have not collected enough zones to determine this number at all in my opinion.
